astro/src/content/docs/lifecycle/migrate-users/bulk/_create-test-application.mdx

```diff
@@ -1,7 +1,7 @@
 import InlineUIElement from 'src/components/InlineUIElement.astro';
 import InlineField from 'src/components/InlineField.astro';
 
-Applications are anything a user can log in to. In FusionAuth there's no differentiation between web applications, SaaS applications, APIs and native apps. To add an application, navigate to <strong>Applications</strong> and click on the <InlineUIElement>Add</InlineUIElement> button (the green plus sign). Give the application a descriptive name like `{props.migration_source_name} application`.
+Applications are anything a user can log in to. In FusionAuth there's no differentiation between web applications, SaaS applications, APIs and native apps. To add an application, navigate to <strong>Applications</strong> and click on the <InlineUIElement>Add</InlineUIElement> button (the green plus sign). Give the application a descriptive name like <code>{props.migration_source_name} application</code>.
```

Comment:
is there a reason we are using code here seemed to work fine with backtick?

Reply:
With one of the changes the backticks were not working.  The tags worked in all situations.